cmd/authelia-scripts/docker.go

```diff
@@ -22,3 +22,12 @@ func (d *Docker) Login(username, password string) error {
 func (d *Docker) Push(tag string) error {
 	return CommandWithStdout("docker", "push", tag).Run()
 }
+
+// Manifest push a docker manifest to dockerhub.
+func (d *Docker) Manifest(tag string, amd64tag string, arm32v7tag string, arm64v8tag string) error {
+	CommandWithStdout("docker", "manifest", "push", "--purge", tag).Run()
+	CommandWithStdout("docker", "manifest", "create", tag, amd64tag, arm32v7tag, arm64v8tag).Run()
+	CommandWithStdout("docker", "manifest", "annotate", tag, arm32v7tag, "--os", "linux", "--arch", "arm").Run()
+	CommandWithStdout("docker", "manifest", "annotate", tag, arm64v8tag, "--os", "linux", "--arch", "arm64", "--variant", "v8").Run()
+	return CommandWithStdout("docker", "manifest", "push", "--purge", tag).Run()
```

Comment:
Why do we push twice?

Reply:
The —purge flag will check to see if a manifest exists locally and remove if so.

It’s a safety net. Second push is where the actual built manifest gets published.

Reply:
Add a comment on the first line then please because it was not obvious.

Reply:
On second thought this will not be required because it hasn't got cached images/manifests due to Travis always spinning up a new host, I'll remove it.